risc0/cargo-risczero/docker/Dockerfile.release

```diff
@@ -5,7 +5,7 @@ RUN apt-get update
 RUN apt-get install -y --no-install-recommends ca-certificates clang curl libssl-dev pkg-config
 RUN curl --proto '=https' --tlsv1.2 --retry 10 --retry-connrefused -fsSL 'https://sh.rustup.rs' | sh -s -- -y
 ENV PATH="/root/.cargo/bin:${PATH}"
-RUN cargo install cargo-risczero
+RUN cargo binstall cargo-risczero
```

Comment:
Can you drop this change for now? We need to install `cargo-binstall` for this to work, and we should pin the version. I think this dockerfile needs more work beyond this as well.